easybuild/easyblocks/p/pybind11.py

```diff
@@ -54,6 +55,13 @@ def extra_options(extra_vars=None):
         extra_vars['download_dep_fail'][0] = True
         return extra_vars
 
+    def configure_step(self):
+        """Avoid that a system Python is picked up when a Python module is loaded"""
+        python_root = get_software_root('Python')
+        if python_root:
+            self.cfg.update('configopts', "-DPYTHON_EXECUTABLE=%s/bin/python" % python_root)
```

Comment:
We should really check if the value already been set beforehand, this is how it's done in the LAMMPS easyblock:
```
option = 'PYTHON_EXECUTABLE'
if "-D%s=" % option not in self.cfg['configopts']:
  self.cfg.update('configopts', '-D%s=%s/bin/python' % (option, python_root))
```

Reply:
We should also log the update

Reply:
fixed in https://github.com/casparvl/easybuild-easyblocks/pull/2

Reply:
Thanks @boegel , I merged it in my branch so this PR is updated with the changes :)